docs/content/Theming/06_browser_color.md

```diff
@@ -0,0 +1,46 @@
+@ngdoc content
+@name Browser Colors
+@description
+## This feature is **mobile** only
+
+![browser color](https://cloud.githubusercontent.com/assets/6004537/18006666/50519c7e-6ba9-11e6-905b-c3751c20549c.png)
+
+Enables browser header coloring with [Material Design Colors](https://material.google.com/style/color.html) and the theming system, <br/> 
+for more info please visit [here](https://developers.google.com/web/fundamentals/design-and-ui/browser-customization/theme-color)
+ 
+## Usage
+### Config phase
```

Comment:
Looking at some of our other docs, we typically capitalize all of the important words in headings.

> Config Phase